mount-helper/install/install.sh

```diff
@@ -294,11 +378,33 @@ fi;
 
 if is_linux LINUX_RED_HAT; then
     check_python3_installed python3
-    if [ "$INSTALL_ARG" != "UNINSTALL" ]; then
+    if [[ $VERSION == 8.8 || $VERSION == 8.9 ]]; then
+        # Define the path to the package list file based on the Ubuntu version
+        PACKAGE_LIST_PATH="packages/rhel/$VERSION/package_list"
+
+        # Check if the package list file exists
+        if [ ! -f "$PACKAGE_LIST_PATH" ]; then
+            exit_err "Package list file '$PACKAGE_LIST_PATH' does not exist"
+        fi
+
+        # Read the package list from the file
+        packages=()
+        while IFS= read -r line; do
+            packages+=("$line")
+        done < "$PACKAGE_LIST_PATH"
+
+        # Install the packages in the defined order
+        if [ "$INSTALL_ARG" != "UNINSTALL" ]; then
+        install_apps packages/epel-release-*.rpm
+        fi
```

Comment:
Any reason why epel is installed seperately than other packages and not included in "package_list"?

Also the path might be causing installation issue,
`install_apps packages/rhel/$VERSION/epel-release-*.rpm`